RN-SafeAreaViewAndroid/Readme.md

```diff
@@ -0,0 +1,50 @@
+SafeAreaView for Android -
+The purpose of SafeAreaView is to render content within the safe area boundaries of a device. It is currently only applicable to iOS devices with iOS version 11 or later. How do we ensure a similar setup for Android devices? Well for one we could add a bit of styling in our SafeAreaView component:-
+
+import React from 'react';
+import { StyleSheet, Text, SafeAreaView } from 'react-native';
+
+const App = () => {
+return (
+<SafeAreaView style={styles.container}>
+<Text>Page content</Text>
+</SafeAreaView>
+);
+}
+
+const styles = StyleSheet.create({
+container: {
+flex: 1,
+},
+});
```

Comment:
- Remove the code from here
- Maintain the structure of the readme, (use bigger fonts for heading and describe the tip), refer to other tips added to the repo